packages/vm/src/state/transientStorage.ts

```diff
@@ -0,0 +1,109 @@
+import { Address } from 'ethereumjs-util'
+
+export interface TransientStorageOps {
+  storage?: Map<Address, Map<string, Buffer>>
+  changesets?: Changeset[][]
+}
+
+export type TStorage = Map<Address, Map<string, Buffer>>
```

Comment:
the map should not be keyed by address, but rather string, two addresses that have the same string representation are not equal in the hashmap